CHANGES.rst

```diff
@@ -96,6 +96,8 @@ astropy.tests
 - Fixed a bug that meant that the data.astropy.org mirror could not be used when
   using --remote-data=astropy. [#6724]
 
+- Support compatibility with new ``pytest-astropy`` plugins. [#6918]
```

Comment:
please add the version number of ``pytest-astropy`` that's being bundled in here

Reply:
The external plugins are actually straight from the master branches of the repositories right now. Can I update and provide this with the next release of `pytest-astropy`?

Reply:
yes, that sounds good. That case it can go to the bottom under "Other Changes"

Reply:
I agree with @bsipocz here - we should tie this to a specific `pytest-astropy` release so affiliated package devs can be sure how to map this onto the "stand-alone" version.